src/init-ceph.in

```diff
@@ -31,11 +31,20 @@ if [ `dirname $0` = "." ] && [ $PWD != "/etc/init.d" ]; then
     SYSTEMD_RUN=""
     ASSUME_DEV=1
 else
-    BINDIR=@bindir@
-    SBINDIR=@prefix@/sbin
-    LIBEXECDIR=@libexecdir@/ceph
-    ETCDIR=@sysconfdir@/ceph
-    ASSUME_DEV=0
+    if [ `dirname $0` = "bin" ]; then
+	BINDIR=bin
+	SBINDIR=bin
+	LIBEXECDIR=../src
```

Comment:
this ../ assumes that your cmake build directory is under ceph/. most of the scripts actually read from CMakeCache.txt to find the right path (see https://github.com/ceph/ceph/blob/master/src/vstart.sh#L27)